test/ToStringTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasTest\Filter;
+
+use Laminas\Filter\ToString;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+use stdClass;
+
+class ToStringTest extends TestCase
+{
+    /** @return list<array{0: mixed}> */
+    public static function returnBasicDataProvider(): array
```

Comment:
Data providers always appear as unused by Psalm when only referenced by the `DataProvider` attribute.

Once all other psalm issues are fixed, these 'PossiblyUnusedMethod' issues can be added to the baseline with `psalm --set-baseline=psalm-baseline.xml`